erpnext/patches/v11_0/make_quality_inspection_template.py

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	frappe.reload_doc('stock', 'doctype', 'item')
+	frappe.reload_doc('stock', 'doctype', 'quality_inspection_template')
+
+	for data in frappe.get_all('Item Quality Inspection Parameter',
+		fields = ["distinct parent"], filters = {'parenttype': 'Item'}):
+		qc_doc = frappe.new_doc("Quality Inspection Template")
+		qc_doc.quality_inspection_template_name = data.parent
```

Comment:
Instead of only item code, name can be like `"QIT/" + item_code`

Reply:
Fixed!